src/resqml2_0_1/IjkGridParametricRepresentation.cpp

```diff
@@ -1451,7 +1451,7 @@ void IjkGridParametricRepresentation::setGeometryAsParametricSplittedPillarNodes
 		// Cumulative
 		resqml2__IntegerHdf5Array* cumulativeLength = soap_new_resqml2__IntegerHdf5Array(gsoapProxy2_0_1->soap, 1);
 		geom->SplitCoordinateLines->ColumnsPerSplitCoordinateLine->CumulativeLength = cumulativeLength;
-		cumulativeLength->NullValue = splitCoordinateLineColumnCumulativeCount[splitCoordinateLineCount - 1] + 1;
+		cumulativeLength->NullValue = getPillarCount();
```

Comment:
Looking at IjkGridExplicitRepresentation.cpp line 437, it looks zero has been chosen as the null value.
Zero looks to me better since it does not require any call to any method and since it should never be a valid value.

for consistency reason, I am going to modify this fix.